cypress/integration/pages/photoGalleryPage/tests.js

```diff
@@ -0,0 +1,63 @@
+import config from '../../../support/config/services';
+import appConfig from '../../../../src/server/utilities/serviceConfigs';
+
+// For testing important features that differ between services, e.g. Timestamps.
+// We recommend using inline conditional logic to limit tests to services which differ.
+export const testsThatAlwaysRun = ({ service, pageType }) => {
+  describe(`No testsToAlwaysRun to run for ${service} ${pageType}`, () => {});
+};
+
+// For testing features that may differ across services but share a common logic e.g. translated strings.
+export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
+  describe(`testsThatFollowSmokeTestConfig to run for ${service} ${pageType}`, () => {
+    it('should render a H1, which displays the headline', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          cy.get('h1').should('contain', body.promo.headlines.headline);
+        },
+      );
+    });
+
+    it('should render a timestamp', () => {
+      cy.request(
+        `${config[service].pageTypes.photoGalleryPage.path}.json`,
+      ).then(({ body }) => {
+        const { lastPublished, firstPublished } = body.metadata;
+        const timeDifferenceMinutes =
+          (lastPublished - firstPublished) / 1000 / 60;
+        const minutesTolerance = 1;
+        cy.get('time')
+          .eq(0)
+          .should('exist')
+          .should('be.visible')
+          .should('have.attr', 'datetime')
+          .should('not.be.empty');
+
+        if (timeDifferenceMinutes > minutesTolerance) {
+          cy.get('time')
+            .eq(1)
+            .should(
+              'contain',
+              appConfig[config[service].name].default.articleTimestampPrefix,
+            );
+        }
+      });
+    });
+
+    it('should render a description for the page, which contains/displays styled text', () => {
```

Comment:
I'm not sure understand the title of this test. 😅Would it make sense to prune the bit about displaying styled text?